pyproject.toml

```diff
@@ -48,7 +48,10 @@ mmcv = "==1.3.8"
 torch = {url = "https://download.pytorch.org/whl/cu111/torch-1.9.1%2Bcu111-cp38-cp38-linux_x86_64.whl"}
 torchvision = {url = "https://download.pytorch.org/whl/cu111/torchvision-0.10.1%2Bcu111-cp38-cp38-linux_x86_64.whl"}
 detectron2 = {url = "https://dl.fbaipublicfiles.com/detectron2/wheels/cu111/torch1.9/detectron2-0.6%2Bcu111-cp38-cp38-linux_x86_64.whl"}
-
+tensorrt = "==10.1.0"
+tensorrt-cu12 = "==10.1.0"
```

Comment:
Do you need to specify these specifically, or are they effectively dependencies of `tensorrt`?

Reply:
@Purg , when I didn't specify them they ended up being a different version and led to conflicts.